projects/cpython3/gen_fuzz.py

```diff
@@ -0,0 +1,47 @@
+"""Generate a fuzz test.
```

Comment:
Also, do you really need to write this in python? 
Even a simple C include will work